balancer-js/src/types.ts

```diff
@@ -78,6 +79,8 @@ export interface ContractAddresses {
   protocolFeePercentagesProvider?: string;
   weightedPoolFactory?: string;
   composableStablePoolFactory?: string;
+
+  [key: string]: string | undefined;
```

Comment:
Just a double check - whats this for?

Reply:
to add new contract addresses without the need to add them on this interface, but I don't think this is a good idea anymore, it opens space for inconsistency, will change that